x/incentives/keeper/keeper_test.go

```diff
@@ -39,8 +40,18 @@ func (s *KeeperTestSuite) ValidateDistributedGauge(gaugeID uint64, expectedFille
 	s.Require().NoError(err)
 	s.Require().Equal(expectedFilledEpoch, gauge.FilledEpochs)
 
+	defaultErrorTolerance := osmomath.ErrTolerance{
```

Comment:
note: adding error tolerance because some of the amount were calculated based on the locks, which truncated down for ex: 2499.123 => 2499 even though the original value was 2500. 

i manually checked that the values weren't off by more than 1